catroid/src/org/catrobat/catroid/ui/fragment/CategoryBricksFactory.java

```diff
@@ -284,7 +284,11 @@
 		);
 		soundBrickList.add(new ChangeVolumeByNBrick(new Formula(defaultValueChangeVolumeBy)));
 
-		soundBrickList.add(new SpeakBrick(BrickValues.SPEAK));
+		if (context.getString(R.string.brick_speak_default_value) != BrickValues.SPEAK) {
+			soundBrickList.add(new SpeakBrick(context.getString(R.string.brick_speak_default_value)));
+		} else {
+			soundBrickList.add(new SpeakBrick(BrickValues.SPEAK));
+		}
```

Comment:
Maybe it's better to remove the variable `BrickValues.SPEAK` entirely and always use `R.string.brick_speak_default_value`?